pages/common/git-imerge.md

```diff
@@ -0,0 +1,24 @@
+# git-imerge
+
+> Perform a merge or rebase between two git branches incrementally.
+> Conflicts between branches are tracked down to pairs of individual commits, to simplify conflict resolution.
+
+- Start imerge-based rebase (checkout the branch to be rebased, first):
+
+`git imerge rebase {{branch_to_rebase_into}}`
+
+- Start imerge-based merge (checkout the branch to merge into, first):
```

Comment:
No, in this case the branch you are on will be modified by the merge, so "into" is more accurate here.